### ISSUE TITLE
autorefresh rooms

### ISSUE THREAD

**Issue by selfsame** [botnikstudios/wodehouse#6]
* poll the server every N seconds
* should be delayed if the mouse is moving or an input is focused

**Comment by thepelkus-too** [comment]
Took a bit of a stab at this this morning, and while I think there's plenty of useable code here, there's still some funkiness e.g. this applies to all inputs, we still don't make the json request if you've deselected a checkbox and it's still focused, everything's hanging out in the global namespace instead of being in, say, an IIFE or something. But just in case someone wants to use it as a jumping off point: https://gist.github.com/thepelkus-too/615f6fd42f4579debf96ce06b39ccac6

**Comment by selfsame** [comment]
Yeah i think we could lose the lodash dep and tighten it up into a single setInterval function with a mouse event that sets a flag or something
